metricq/agent.py

```diff
@@ -128,14 +127,14 @@ def __init__(
             timeout=connection_timeout,
             connection_name="management connection",
         )
-        self._management_channel = None
+        self._management_channel: Optional[aio_pika.Channel] = None
 
-        self.management_rpc_queue = None
+        self.management_rpc_queue: Optional[aio_pika.Queue] = None
 
         self._management_broadcast_exchange = None
         self._management_exchange = None
 
-        self._rpc_response_handlers = dict()
+        self._rpc_response_handlers: Dict[str, Tuple[Any, bool]] = dict()
```

Comment:
```suggestion
        self._rpc_response_handlers: Dict[str, Tuple[Callable, bool]] = dict()
```

How about this?

> A plain `Callable` is equivalent to `Callable[..., Any]`

We actually take any number of keyword arguments and return anything - including nothing (`None`).